filewriter_v2/README.md

```diff
@@ -0,0 +1,388 @@
+| [beam_center_x](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-beam-center-x-field) | NX_FLOAT | The x position of the direct beam on the detector. |
+| beam_center_x@units | NX_LENGTH | `"pixel"` |
+| [beam_center_y](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-beam-center-y-field) | NX_FLOAT | The y position of the direct beam on the detector. |
+| beam_center_y@units | NX_LENGTH | `"pixel"` |
+| [bit_depth_readout](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-bit-depth-readout-field) | NX_INT | Bit depth of the internal readout. |
+| [count_time](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-count-time-field) | NX_FLOAT | Exposure time per image. |
+| count_time@units | NX_TIME | `"s"` |
+| [countrate_correction_applied](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-countrate-correction-applied-field) | NX_BOOLEAN | Indicates whether countrate correction was applied. |
+| [countrate_correction_lookup_table](https://manual.nexusformat.org/classes/base_classes/NXdetector.html#nxdetector-countrate-correction-lookup-table-field) | NX_NUMBER[] | The countrate_correction_lookup_table defines the LUT used for count-rate correction. It maps a measured count `c` to its corrected value `countrate_correction_lookup_table[c]`. |
+| [depends_on](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-depends-on-field) | NX_CHAR | NeXus path to the detector positioner axis that most directly supports the detector. In the case of a single-module detector, the detector axis chain may start here. |
+| [description](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-description-field) | NX_CHAR | Detector type and model information. |
+| [detector_readout_time](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-detector-readout-time-field) | NX_FLOAT | Readout dead time between consecutive detector frames. |
+| detector_readout_time@units | NX_TIME | `"s"` |
+| [distance](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-distance-field) | NX_FLOAT | Distance from the sample to the beam center on the detector. |
+| distance@units | NX_LENGTH | `"m"` |
+| [flatfield_applied](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-flatfield-applied-field) | NX_BOOLEAN | Indicates whether a flatfield has been applied. |
+| [frame_time](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-frame-time-field) | NX_FLOAT | Time interval between start of image acquisitions. This defines the speed of data collection and is the inverse of the frame rate, the frequency of image acquisition. |
```

Comment:
Time interval between the start ...

Reply:
fixed in [34890fd](https://github.com/soph-dec/documentation/pull/1/commits/34890fdc03430a5b054a735918d0593e38a8e157)